docs/ruleset_functions/baseline_systems/is_baseline_system_4.md

```diff
@@ -25,9 +25,9 @@
 
 
 ## Logic:    
-- Set is_baseline_system_4 = Not_Sys_4: `is_baseline_system_4 = "Not_Sys_4"` 
+- Set is_baseline_system_4 = Not_Sys_4: `is_baseline_system_4 = "Not_Sys_4"`
 - Create an object associated with the hvac system: `hvac_b = hvac_b.id`  
-- Check that there is no preheat system, if there is none then carry on: `if len(hvac_b.preheat_system) == Null or hvac_b.preheat_system[0].heating_system_type == "NONE" :`   
+- Check that there is no preheat system, if there is none then carry on: `if hvac_b.preheat_system == "NONE" or hvac_b.preheat_system[0].heating_system_type == "NONE" :`   
```

Comment:
will need to update `hvac_b.preheat_system[0].heating_system_type` to `hvac_b.preheat_system.heating_system_type`

Reply:
great catch. Fixed.